lib/hash.cc

```diff
@@ -221,20 +201,19 @@ hash_last(hash_table * hid)
  *  list.
  */
 void
-hash_remove_link(hash_table * hid, hash_link * hl)
-{
+hash_table::hash_remove_link(hash_link *hl) {
```

Comment:
Please keep the Squid coding style with brackets on a separate line.
```suggestion
hash_table::hash_remove_link(hash_link *hl)
{
```